zbm-builder.sh

```diff
@@ -95,8 +102,8 @@ fi
 
 # If no config is specified, use in-tree default
 if ! [ -r ./config.yaml ]; then
-  BUILD_ARGS+=( "-c" "/zbm/etc/zfsbootomenu/config.yaml" )
+  BUILD_ARGS+=( "-c" "/zbm/etc/zfsbootmenu/config.yaml" )
 fi
 
 # Make `/build` the working directory so relative paths in a config file make sense
-"${PODMAN}" run --rm -v ".:/build" -w "/build" "${BUILD_TAG}" "${BUILD_ARGS[@]}"
+"${PODMAN}" run --rm "${VOLUME_ARGS[@]}" -v "${PWD}:/build" -w "/build" "${BUILD_TAG}" "${BUILD_ARGS[@]}"
```

Comment:
Instead of having two different ways to specify volumes (`$VOLUME_ARGS` and then hard-coded), you could instead push all volumes to `$VOLUME_ARGS`. The default value of `-v "${PWD}:/build"` can be pushed to the array when it's defined.